src/aioquic/tls.py

```diff
@@ -1571,13 +1575,21 @@ def _client_handle_certificate_verify(self, input_buf: Buffer) -> None:
 
         # check signature
         try:
-            self._peer_certificate.public_key().verify(
-                verify.signature,
-                self.key_schedule.certificate_verify_data(
-                    b"TLS 1.3, server CertificateVerify"
-                ),
-                *signature_algorithm_params(verify.algorithm),
-            )
+            if isinstance(self._peer_certificate.public_key(), ed25519.Ed25519PublicKey):
+                self._peer_certificate.public_key().verify(
```

Comment:
Same as below, adjusting `signature_algorithm_params` would avoid this repeated code.